spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/src/main/java/org/springframework/cloud/stream/binder/kafka/config/KafkaMessageChannelBinderConfiguration.java

```diff
@@ -33,9 +34,13 @@
 @ConfigurationProperties(prefix = "spring.cloud.stream.binder.kafka")
 public class KafkaMessageChannelBinderConfiguration {
 
-	private String zkAddress;
+	private String[] zkAddress;
```

Comment:
Let's make it zkNodes. I've tried to keep the old values from xd, but at this point it makes very little sense.